build.ps1

```diff
@@ -28,20 +28,29 @@ function CrossGen([string] $runtime, [bool] $isSelfContained, [string] $publishT
     $regular = Join-Path $publishTarget "regular"
     $crossGen = "$publishTarget\download\crossgen\crossgen.exe"
 
-    #DownloadNupkg "https://dotnet.myget.org/F/dotnet-core/api/v2/package/runtime.$runtime.microsoft.netcore.jit/2.1.0-preview2-26316-09" @("runtimes\$runtime\native\clrjit.dll")  @("$publishTarget\download\clrjit")
-    #DownloadNupkg "https://dotnet.myget.org/F/dotnet-core/api/v2/package/runtime.$runtime.Microsoft.NETCore.Runtime.CoreCLR/2.1.0-preview2-26316-09"  @("tools\crossgen.exe")  @("$publishTarget\download\crossgen")
+	#$symbolsPath = Join-Path $publishTarget "Symbols"
+    #new-item -itemtype directory -path $symbolsPath
 
-    DownloadNupkg "https://www.nuget.org/api/v2/package/runtime.$runtime.Microsoft.NETCore.Jit/2.0.5" @("runtimes\$runtime\native\clrjit.dll")  @("$publishTarget\download\clrjit")
-    DownloadNupkg "https://www.nuget.org/api/v2/package/runtime.$runtime.Microsoft.NETCore.Runtime.CoreCLR/2.0.5"  @("tools\crossgen.exe")  @("$publishTarget\download\crossgen")
+    DownloadNupkg "https://dotnet.myget.org/F/dotnet-core/api/v2/package/runtime.$runtime.Microsoft.NETCore.Jit/2.1.0-rtm-26528-02" @("runtimes\$runtime\native\clrjit.dll")  @("$publishTarget\download\clrjit")
+    DownloadNupkg "https://dotnet.myget.org/F/dotnet-core/api/v2/package/runtime.$runtime.Microsoft.NETCore.Runtime.CoreCLR/2.1.0-rtm-26528-02"  @("tools\crossgen.exe")  @("$publishTarget\download\crossgen")
     DownloadNupkg "https://www.nuget.org/api/v2/package/Microsoft.Build.Tasks.Core/15.1.1012" @("lib\netstandard1.3\Microsoft.Build.Tasks.Core.dll")  @("$selfContained")
     DownloadNupkg "https://www.nuget.org/api/v2/package/Microsoft.Build.Utilities.Core/15.1.1012" @("lib\netstandard1.3\Microsoft.Build.Utilities.Core.dll")  @("$selfContained")
+    if ($runtime -eq "win-x86") {
+        DownloadNupkg "https://dotnet.myget.org/F/aspnetcore-dev/api/v2/package/Microsoft.AspNetCore.AspNetCoreModuleV2/2.1.0-a-oob-2-1-oob-16914" @("contentFiles\any\any\x86\aspnetcoreV2.dll", "contentFiles\any\any\x86\aspnetcoreV2_inprocess.dll") @("$privateSiteExtensionPath\ancm", "$privateSiteExtensionPath\ancm")
+    } else {
+        DownloadNupkg "https://dotnet.myget.org/F/aspnetcore-dev/api/v2/package/Microsoft.AspNetCore.AspNetCoreModuleV2/2.1.0-a-oob-2-1-oob-16914" @("contentFiles\any\any\x64\aspnetcoreV2.dll", "contentFiles\any\any\x64\aspnetcoreV2_inprocess.dll") @("$privateSiteExtensionPath\ancm", "$privateSiteExtensionPath\ancm")
+    }
+    DownloadNupkg "https://www.nuget.org/api/v2/package/Microsoft.Web.Xdt/2.1.2" @("lib\net40\Microsoft.Web.XmlTransform.dll")  @("$publishTarget\download\xdt")
```

Comment:
This download/extraction can also be removed and we can simplify the transformation (by removing the custom implementation) if we install the `dotnet-xdt` global tool:
`dotnet tool install -g dotnet-xdt --version 2.1.0-rc.1`